### PR TITLE
Add `New Featured Artist: ptar124` news post

### PR DESCRIPTION
**Pull request by pishifat**
!!!

### CODE REVIEW

File: news/2024/2024-03-31-new-featured-artist-ptar124.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+layout: post
+title: "New Featured Artist: ptar124"
+date: 2024-03-31 20:50:00 +0000
+---
+
+osu!taiko veteran **ptar124** has gone from digital drumming to... well, still digital drumming, but with their own music as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/409/header.jpg)
+
+**ptar124** is a name anyone in the tournament scene should recognize. Their trance/hardcore tunes have been highlighted in tournaments like [GTS](https://osu.ppy.sh/wiki/en/Tournaments/GTS), [AxS](https://osu.ppy.sh/community/forums/topics/1474871?n=1), [NTJ](https://osu.ppy.sh/community/forums/topics/1838639?n=1), and most recently our very own [osu!taiko World Cup](https://osu.ppy.sh/wiki/Tournaments/TWC/2024) with [aqua vitae](https://osu.ppy.sh/beatmapsets/2155621) and [Ad infinitum ~ein sof~](LINK)!
```
Comment (Nivalyx) [inline news/2024/2024-03-31-new-featured-artist-ptar124.md:11]:
```suggestion
**ptar124** is a name anyone in the tournament scene should recognize. Their trance/hardcore tunes have been highlighted in tournaments like [GTS](https://osu.ppy.sh/wiki/en/Tournaments/GTS), [AxS](https://osu.ppy.sh/community/forums/topics/1474871?n=1), [NTJ](https://osu.ppy.sh/community/forums/topics/1838639?n=1), and most recently our very own [osu!taiko World Cup](https://osu.ppy.sh/wiki/Tournaments/TWC/2024) with [aqua vitae](https://osu.ppy.sh/beatmapsets/2155621) and [Ad infinitum \~ein sof\~](LINK)!
```

File: news/2024/2024-03-31-new-featured-artist-ptar124.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+In case you're looking for some inspiration, here's two instances of **ptar124**'s digital drumming in action, both of which debuted in this year's osu!taiko World Cup:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/409/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### ptar124 & SiLiS - Ad infinitum ~ein sof~
```
Comment (Nivalyx) [inline news/2024/2024-03-31-new-featured-artist-ptar124.md:23]:
```suggestion
### ptar124 & SiLiS - Ad infinitum \~ein sof\~
```

File: news/2024/2024-03-31-new-featured-artist-ptar124.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Check out [the map from the video above](LINK) hosted by [frz](https://osu.ppy.sh/users/6956922)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/409/Songs/Viatores%20Nebulae%20(ptar124%20%26%20SiLiS)%20-%20Ad%20infinitum%20%7Eein%20sof%7E.mp3">
+</audio>
+
+### ptar124 - AmbidextrouS ~The Alchemist of The Left x The Sage of The Right~
```
Comment (Nivalyx) [inline news/2024/2024-03-31-new-featured-artist-ptar124.md:31]:
```suggestion
### ptar124 - AmbidextrouS \~The Alchemist of The Left x The Sage of The Right\~
```

**Review by Hiviexd — APPROVED** [review]
taiko eating good

**Pull request merged**